tests/test_main.py

```diff
@@ -627,87 +632,89 @@ def is_annotation(node):
     return False
 
 
-@unittest.skipUnless(
-    os.getenv('EXECUTING_SLOW_TESTS'),
-    'These tests are very slow, enable them explicitly',
-)
-class TestFiles(unittest.TestCase):
```

Comment:
One advantage of the class is automatically applying the skip to both tests. See https://docs.pytest.org/en/7.1.x/getting-started.html#group-multiple-tests-in-a-class

Reply:
ohh, I got a problem as long as the class derived from `unittest.TestCase`, but a normal class should work too.